.ls-lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+# ls-lint configuration file. More information on the file format can be found on https://ls-lint.org/
+ls:
+  # all Markdown files are like CODE_OF_CONDUCT.md
+  .md: screamingsnakecase
+
+  # anything under this folder but not the folder itself
+  vale/styles/jargonLint/:
+    .dir: snake_case          # all subdirectories must comply this rule
+    .OLD: screamingsnakecase  # the LICENSE.OLD files
+    .yml: pascalcase
+    .yaml: pascalcase
+    .txt: pascalcase
+    .good.txt: pascalcase
+    .bad.txt: pascalcase
```

Comment:
I may have misunderstood something, aren't the rules here supposed to be for the entire repository?

Reply:
They/we could

Here I'm enforcing the rules on whole repository only for Markdown files.

Then I'm enforcing the rules for files in vale/styles/jargonLint

We could enforce them for other format at repository level, but then I would have to add files to ignore, because .yamllint.yml, .ls-lint.yml, .markdownlint.yml doesn't comply.

I tried to keep things simple and avoid adding exception in this PR while other PR were currently opened. But we could, sure.

Here I only added the rules we needed for vale rules

Reply:
Okay, since most exceptions will likely be in root or other directories, what if we made rules apply for ``vale`` and ``sources``? That way, most of the files that the linter needs to check are covered.

Reply:
We can for sure.

But I will wait for files to be merged then.

Or we could merge like, this, then either you rebase and get ls-lint in your branches so you enforce the rules in your PR

It's uneasy to do it with multiple  pending PRs

Reply:
Okay let's merge #46 first.

Reply:
OK 👍

Reply:
The following one was merged.

- #46

I think we can merge this one now.

What do you think @alexvonme